.github/workflows/lint.yaml

```diff
@@ -12,3 +18,14 @@ jobs:
 
     - name: Run linting script
       run: ${PWD}/hack/ci/link-check.sh
+
+    # Note: might be worth pivoting to https://github.com/marketplace/actions/create-an-issue
+    # if we're interested in housing a custom template file.
+    - name: Create an issue if the lint job fails
+      uses: nashmaniac/create-issue-action@v1.1
+      if: failure() && github.event_name != 'pull_request' && github.ref == 'refs/heads/master'
```

Comment:
I think this conditional is right (basically don't run this when it's on a pull_request event trigger (e.g. it's either a master branch push event or a schedule event)?